store/snapshots/manager.go

```diff
@@ -76,12 +77,19 @@ func NewManager(store *Store, opts types.SnapshotOptions, multistore types.Snaps
 	if extensions == nil {
 		extensions = map[string]types.ExtensionSnapshotter{}
 	}
+	var snapAnnouncer types.SnapshotAnnouncer
+	if v, ok := multistore.(types.SnapshotAnnouncer); ok {
+		snapAnnouncer = v
+	} else {
+		snapAnnouncer = noopSnapshotAnnouncer{}
+	}
```

Comment:
we can just do var `snapAnnouncer types.SnapshotAnnouncer  = noopSnapshotAnnouncer{}` in the declaration?

Reply:
np, but I can not push to this branch anymore due to permission updates